### ISSUE TITLE
'debug:android' failing

### ISSUE THREAD

**Issue by kenmanheimer** [SpiderOak/SpiderOakMobileClient#298]
After a few days of test development I returned to do a 'grunt debug:android' and was blocked by the following traceback.  (While this may, somehow, be specific to my setup, I'm unable to unravel that at the moment. Heading in that direction, I first tried 'npm install', and was then blocked by a backwards-incompatibility with a new underscore version, but that's a separate issue.)

```
Running "shell:debug_android" (shell) task
Warning: Command failed: [Error: Script "/Volumes/MacCrisp/Users/klm/Projects/SpiderOak/mobile/HTML5Client/SpiderOakMobileClient/.cordova/hooks/before_prepare/version.js" exited with non-zero status code. Aborting. Output: 
module.js:340
    throw err;
          ^
Error: Cannot find module 'elementtree'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /Volumes/MacCrisp/Users/klm/Projects/SpiderOak/mobile/HTML5Client/SpiderOakMobileClient/.cordova/hooks/before_prepare/version.js:6:10
    at Object.<anonymous> (/Volumes/MacCrisp/Users/klm/Projects/SpiderOak/mobile/HTML5Client/SpiderOakMobileClient/.cordova/hooks/before_prepare/version.js:16:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
]
 Use --force to continue.
```
